tests/model_selection/test_incremental.py

```diff
@@ -244,17 +245,18 @@ def _test_search_basic(decay_rate, c, s, a, b):
         assert all(search.cv_results_["test_score"] >= 0)
         assert all(search.cv_results_["rank_test_score"] >= 1)
     assert all(search.cv_results_["partial_fit_calls"] >= 1)
-    assert len(np.unique(search.cv_results_["model_id"])) == len(
-        search.cv_results_["model_id"]
+    assert len(np.unique(search.cv_results_["estimator_id"])) == len(
+        search.cv_results_["estimator_id"]
```

Comment:
Changing `model_id` to `estimator_id` will break backwards compatibility.

Reply:
The renaming from `model_id` to `estimator_id` should be removed from this PR. That's for another PR.